src/sickle.ts

```diff
@@ -268,7 +268,7 @@ class Annotator {
 
   private visitEnum(node: ts.EnumDeclaration) {
```

Comment:
odd to have `visit` and `writeNode` with a skipComments flag, and not visitEnum. Can the flag be added to visitEnum and at callsite in visit - pass true.